CemrgApp/CMakeExternals/VMTK.cmake

```
@@ -34,8 +34,8 @@ if(MITK_USE_VMTK)
       )
     endif()
 
-    set(VMTK_GIT_REPOSITORY "https://github.com/vmtk/vmtk.git" CACHE STRING "The git repository for cloning VMTK")
-    set(VMTK_GIT_TAG "v1.4.0" CACHE STRING "The git tag/hash to be used when cloning from VMTK_GIT_REPOSITORY")
+    set(VMTK_GIT_REPOSITORY "https://github.com/CemrgDevelopers/vmtk.git" CACHE STRING "The git repository for cloning VMTK")
```

```suggestion
    set(VMTK_GIT_REPOSITORY "https://github.com/OpenHeartDevelopers/vmtk.git" CACHE STRING "The git repository for cloning VMTK")
```

`CemrgDevelopers` still works but redirects to `OpenHeartDevelopers`. The org must have been renamed at some point, so let's use the most recent name